sky/resources.py

```diff
@@ -1008,7 +1008,7 @@ def _try_validate_labels(self) -> None:
                         f'Label rejected due to {cloud}: {err_msg}'
                     ])
                     break
-        if len(invalid_table.rows) > 0:
+        if invalid_table.rows:
```

Comment:
same, lets check if the `rows` attribute can be converted this way.

Reply:
`rows` is a `list`, and the answer is yes